call_host.sh

```diff
@@ -46,18 +102,41 @@ if [ ! -d "$CALL_HOST_DIR" ]; then
 	echo "Warning: could not create specified dir CALL_HOST_DIR $CALL_HOST_DIR. disabling"
 	export CALL_HOST_STATUS=disable
 fi
-# ensure the pipe dir is bound
-export APPTAINER_BIND=${APPTAINER_BIND}${APPTAINER_BIND:+,}${CALL_HOST_DIR}
+
+# helper: add value to a PATH-like variable only if not already present
+add_path_unique(){
+	# args: varname value [sep]
+	varname="$1"; val="$2"; sep="${3:-:}"
+	# retrieve current value portably
+	cur="$(eval "printf '%s' \"\${${varname}:-}\"")"
```

Comment:
maybe should be a function, since it's used repeatedly (also for `$VAR_TO_VALIDATE` above)

Reply:
OK, will do